src/scss/components/_gallery-item.scss

```diff
@@ -0,0 +1,62 @@
+/**
+  * @parent gaiden_css.components
+  * @stylesheet gaiden_css.components.gallery-item.css Gallery Item
+  *
+  * ## Services
+  * @demo demo/gaiden-css/components/gallery-item/demo.html
+**/
+.gallery-item {
```

Comment:
`gallery-item` is not what this item is it. It's more like a `card`, or `framed picture`, or even a `thumbnail`.

When I see the name `gallery`, I imagine a picture gallery and this is not the case.

Even to use `gallery-item`, `item` make me think that the item is part of gallery, what is not the case too.